tools/src/main/java/org/apache/kafka/tools/VerifiableProducer.java

```diff
@@ -144,6 +153,15 @@ private static ArgumentParser argParser() {
                 .metavar("CONFIG_FILE")
                 .help("Producer config properties file.");
 
+        parser.addArgument("--message-create-time")
```

Comment:
I guess another way would be to let this configuration be a delta which is added to the the current time. That way we wouldn't have a lot of messages with the same timestamp, which might be a little uncommon in practice.

Reply:
Messages are assigned create times _starting_ from this value. The time is incremented on every send by the interval between the sends. So they all won't have the same create time.

Reply:
Ah, ok. That works.